README.md

```diff
@@ -477,8 +478,96 @@ While this approach may seem odd. It provides a way to deliver drivers to immuta
 [Mellanox OFED and NV Peer Memory driver container](https://github.com/Mellanox/ofed-docker)
 
 ## Upgrade
-Network operator provides limited upgrade capabilities which require additional
-manual actions if a containerized OFED driver is used.
-Future releases of the network operator will provide automatic upgrade flow for the containerized driver.
-
 Check [Upgrade section in Helm Chart documentation](deployment/network-operator/README.md#upgrade) for details.
+
+## Externally Provided Configurations For Network Operator Sub-Components
```

Comment:
@e0ne PTAL

we can either  keep this section here or move it to its own document under /docs/
as its an advanced use case.